packages/payment/src/PaymentService.ts

```diff
@@ -0,0 +1,64 @@
+import { injectable } from 'inversify';
+import * as express from 'express';
+/* import * as mongoSanitize from 'express-mongo-sanitize';
+import * as moment from 'moment';
+import { WishCardRepository } from '@donategifts/wishcard'; */
+import { IStripeIntent /* IDonationInfo */, IDonationHook } from '@donategifts/common';
+import { StripeService } from './StripeService';
+import { PaypalService } from './PaypalService';
+import { handleDonation } from './helper/paymentHelper';
+
+@injectable()
+export class PaymentService {
+	constructor(
+		private stripeService = new StripeService(),

```

Comment:
we don't do it the JS way anymore, we use inversify to handle our injections

we inject the services with the ```@inject``` decorator to create single instances of the classes used
```ts
@inject(StripeService) private stripeService: StripService,
@inject(PaypalService) private paypalService: PaypalService
```